### DISCUSSION TITLE
Descending sort not correct

### DISCUSSION THREAD

**Discussion by aagha** [saulpw/visidata#2350] [Q&A]
I'm sorting data in a file containing financial transactions, at it seems that the Debit column isn't properly sorting:

![image](https://github.com/saulpw/visidata/assets/5522103/431a7ecf-caf8-4552-a6c3-53dbd304007f)

Any idea why?

**Comment by anjakefala** [comment]
Hi!

I think it is because the debit column is not typed. Use `%` to type it as a float column, and it should sort correctly.
